parsec/core/cli/bootstrap_organization.py

```diff
@@ -55,27 +63,25 @@ async def _bootstrap_organization(
         )
         if not answer:
             raise SystemExit("Bootstrap aborted")
-        sequester_verify_key = SequesterVerifyKeyDer(sequester_verify_key.read_bytes())
+        sequester_vrf_key = SequesterVerifyKeyDer(sequester_verify_key.read_bytes())
 
-    if not human_label:
-        human_label = await aprompt("User fullname")
-    if not human_email:
-        human_email = await aprompt("User email")
+    human_label: str = human_label or await aprompt("User fullname")
+    human_email: str = human_email or await aprompt("User email")
     human_handle = HumanHandle(email=human_email, label=human_label)
     if not device_label:
-        device_label_raw = await aprompt("Device label", default=platform.node())
-        device_label = DeviceLabel(device_label_raw)
+        device_label_raw: str = await aprompt("Device label", default=platform.node())
+        dev_label = DeviceLabel(device_label_raw)
```

Comment:
Why rename ?

Reply:
Same as above, `mypy` don't like to pass from a `str` to a `DeviceLabel` for the same variable